app/store/reducers.js

```diff
@@ -0,0 +1,9 @@
+import {combineReducers} from 'redux'
+
+import fetchPlayersReducer from './fetchPlayersReducer'
+import fetchGamesReducer from './fetchGamesReducer'
+
+export default combineReducers({
```

Comment:
Could probably name this export, or export as something other than combineReducers. At first glance confused me as I though you were importing the react-redux function in `app.js`.